### ISSUE TITLE
Cross validation when using dummy observations

### ISSUE THREAD

**Issue by ACharsley** [James-Thorson/VAST#98]
Hi,
I have followed your method for making predictions as outlined here: https://github.com/James-Thorson/VAST/issues/76. I have a very large set of dummy observations that I am trying to predict by specifying them as PredTF_i=1. However, I would now like to cross-validate this model using the Crossvalidate_Fn function. Of course this means that the function tries to cross-validate using dummy observations as well as true observations. Is there a way to cross-validate this model? Or would I only be able to cross-validate a model that doesn't contain these dummy observations?

**Comment by James-Thorson** [comment]
Hmm.  Why do you have the dummy observations -- is it to indicate to the model that you should predict in years without any observations besides those "dummy" observations?

**Comment by James-Thorson** [comment]
OK, so in this case you can edit Crossvalidate_Fn to replace `Save$Report$pred_jnll` as the measure of the predictive log-likelihood with `sum(Save$Report$LogProb1_i[Which] + Save$Report$LogProb2_i[Which])` where `Which` indicates the subset of observations that you want to include in that crossvalidation partition (i.e., after excluding the dummy observations).

**Comment by ACharsley** [comment]
Thanks, but wouldn't the cross validation still incorporate "false" data (the b_i I had to assign to the dummy observations)?

**Comment by James-Thorson** [comment]
You'll have to input `Which` such that it excludes all but a given partition and also excludes the dummy observations.  I don't have any way to do this built in, but that is one backdoor you could use, there's probably others too.

**Comment by ACharsley** [comment]
Yes, but wouldn't the values of `Save$Report$LogProb1_i[Which]` and `Save$Report$LogProb2_i[Which]` be influenced by the particular observations (some of them being false) selected for the model by `PredTF_i` at each fold?
